bazelrio/dependencies/scripts/deps.bzl

```diff
@@ -0,0 +1,20 @@
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+load("@rules_python//python:pip.bzl", "pip_install")
+
+def setup_scripts_dependencies():
+    maven_install(
+        artifacts = [
+            "com.hierynomus:sshj:0.32.0",
+            "me.tongfei:progressbar:0.9.2",
+            "net.sourceforge.argparse4j:argparse4j:0.9.0",
+            "org.slf4j:slf4j-nop:1.7.32",
```

Comment:
If I'm reading [this](http://slf4j.org/log4shell.html) right, this should still be safe, right?

Reply:
I think so, since we're using the no-op implementation and not a log4j implementation